src/project-2.js

```diff
@@ -47,61 +76,77 @@ const isPrime = (num) => {
 };
 
 const returnFirst = (arr) => {
+  return arr[0];
   // return the first item from the array
 };
 
-const returnLast = (arr) => {
+const returnLast = arr => arr.pop();
```

Comment:
`arr.pop()` removes the last item from the array.  You should do: `arr[arr.length - 1];` . This just reads the last item in the array.